lib/roo_on_rails/railties/http.rb

```diff
@@ -17,10 +17,12 @@ class HTTP < Rails::Railtie
           ::Rack::Timeout
         )
 
+        middleware_to_insert_before = Rails::VERSION::MAJOR < 4 ? ::ActionDispatch::Cookies : ::Rack::Head 
```

Comment:
Intention is instead of using `ActionDispatch::Cookies` using `Rack::Head` which is common in `Rails 4`, `Rails 5` and `Rails 5 Api only` but not in `Rails 3`. But we have `ActionDispatch::Cookies` for Rails 3.